composer.json

```diff
@@ -107,8 +107,6 @@
       "@build"
     ],
     "build-db": [
-      "drush @local.mysiteurl sql-drop -y",
```

Comment:
please revert this and add `[ "$CI" == "0" ] && drush @local.mysiteurl sql-drop -y`, `CI` env variable is populated in CI